src/nifake/metadata/config.py

```diff
@@ -5,6 +5,7 @@
     'c_function_prefix': 'niFake_',
     'driver_name': 'NI-FAKE',
     'session_description': 'An NI-FAKE session to a fake MI driver whose sole purpose is to test nimi-python code generation',
+    'session_name': 'vi',
```

Comment:
Is this the name the generator will use for the member that stores the session handle?
Anything else?

Reply:
It's not, it's the parameter name used by the C API as the session handle.